indico/modules/rb/templates/emails/reservations/change_state_email_to_vc_support.txt

```diff
@@ -0,0 +1,6 @@
+{% extends 'rb/emails/reservations/base_email_to_vc_support.txt' %}
+
+{% block body -%}
+A booking on {{ reservation.room.full_name }} was modified and the changes require approval.
```

Comment:
In the other two email templates the word "modified" is in capital letters, I don't know what's the correct way but I think we should be consistent.

Reply:
I'd really like those AWFUL CAPS to GO AWAY ;)